pkg/cluster/context.go

```diff
@@ -178,6 +178,18 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// try to remove the kind kube config file generated by "kind create cluster"
+	err = os.Remove(c.KubeConfigPath())
+	if err != nil {
+		fmt.Printf("Tried to remove %s but recieved error: %s\n", c.KubeConfigPath(), err)
```

Comment:
I think it this should be logged instead of just printed.

Reply:
Yeah, I think you're right. Will fix.

Reply:
What level tho, warning?

Reply:
Warning 👍

Reply:
I would think it deserves an error

Reply:
What Ben said lol

Reply:
I think cleanup failures are generally not treated as critical in kind, but the user / logging should be informed. we can revisit it later if usage proves that to be incorrect.